src/EnergyPlus/PollutionModule.hh

```diff
@@ -59,13 +59,76 @@
 
 namespace EnergyPlus {
 
-namespace PollutionModule {
+namespace Pollution {
 
-    enum class PollFactor
+    enum class Pollutant
     {
         Invalid = -1,
-        Elec,
-        NatGas,
+        CO2,
+        CO,
+        CH4,
+        NOx,
+        N2O,
+        SO2,
+        PM,
+        PM10,
+        PM2_5,
+        NH3,
+        NMVOC,
+        Hg,
+        Pb,
+        Water,
+        NuclearHigh,
+        NuclearLow,
+        Num
+    };
+
+    constexpr std::array<std::string_view, (int)Pollutant::Num> poll2Names = {
```

Comment:
Made these into `constexpr std::array<X>` to enable use of loops rather than copy-paste.